its/projects/ReproAzureFunctions/NuGet.Config

```
@@ -2,8 +2,14 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
-= </packageSources>
+    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/v3/nuget/index.json" protocolVersion="3" />
+  </packageSources>
+  <packageSourceCredentials>
+    <Repox_x0020_Artifactory_x0020__x0028_proxy_x0029_>
```

We're doing this refactoring in another PR that touches the main nuget.config
```suggestion
    <add key="Repox" value="https://repox.jfrog.io/artifactory/api/nuget/v3/nuget/index.json" protocolVersion="3" />
  </packageSources>
  <packageSourceCredentials>
    <Repox>
```

---

To keep it simple. Makes sense.